release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -568,6 +568,80 @@ For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-
 [id="ocp-4-11-monitoring"]
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
+
+==== Monitoring stack components and dependencies
+
+Updates to versions of monitoring stack components and dependencies include the following:
+
+* Alertmanager to 0.24.0
+* kube-state-metrics to 2.5.0
+* node-exporter to 1.3.1
+* prom-label-proxy to 0.4.0
+* Prometheus to 2.36.2
+* Prometheus adapter to 0.9.1
+* Prometheus operator to 0.57.0
+* Thanos to 0.26.0
```

Comment:
from https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-monitoring-stack-components-and-dependencies
```
node-exporter to 1.3.1
prom-label-proxy to 0.4.0
Prometheus adapter to 0.9.1
```
these versions are not changed since 4.10, do we need to mention it again?

Reply:
Removed these from the list